src/utils/auth/oidc.ts

```diff
@@ -0,0 +1,51 @@
+import Consola from 'consola';
+import { OAuthConfig } from 'next-auth/providers/oauth';
+
+import adapter from './adapter';
+import { env } from '~/env';
+
+type Profile = {
+  sub: string
+  name: string
+  email: string
+  groups: string[]
+  preferred_username: string
+  email_verified: boolean
+}
+
+const provider: OAuthConfig<Profile> = {
+  id: "oidc",
+  name: env.AUTH_OIDC_CLIENT_NAME,
+  type: "oauth",
+  clientId: env.AUTH_OIDC_CLIENT_ID,
+  clientSecret: env.AUTH_OIDC_CLIENT_SECRET,
+  wellKnown: `${env.AUTH_OIDC_URI}/.well-known/openid-configuration`,
```

Comment:
Is this a common standard?

Reply:
yes, the suffix is part of specification, but the env variable could technically point to directly to the configuration as it's used nowhere else